packages/maps/src/Client.ts

```diff
@@ -17,22 +17,22 @@ export class Maps {
    * @param options
    */
   public async instantiateMapFrom(options: MapOptions) {
-    const { sql, dataset, vector_extent, vector_simplify_extent } = options;
+    const { sql, dataset, vector_extent = 2048, vector_simplify_extent = 2048, metadata = {} } = options;
 
     if (!(sql || dataset)) {
       throw new Error('Please provide a dataset or a SQL query');
     }
 
     const mapConfig = {
       layers: [{
-        type: 'cartodb',
+        type: 'mapnik',
         options: {
           sql: sql || `select * from ${dataset}`,
-          vector_extent: vector_extent || 2048,
-          vector_simplify_extent: vector_simplify_extent || 2048
+          vector_extent,
+          vector_simplify_extent,
+          metadata
         }
-      }],
-      version: '1.3.1'
+      }]
```

Comment:
Are you sure we don't need 'version' field anymore on mapConfig? What's the behavior without it / latest?

Reply:
It is `1.0.0` as per docs. I removed it because I saw that CARTO-VL didn't include that in the request. But thinking it right now, I think we should keep it just for safety.